terraform-random-pet/README.md

```diff
@@ -0,0 +1,102 @@
+# terraform-random-pet
+
+This is a template child module.
+
```

Comment:
Let's put a markdown comment here that specifies to delete these lines after reading them and copying the template for use: 

```suggestion
<!-- README TEMPLATE: AFTER READING THE BELOW SECTION, DELETE THE BELOW SECTION AND REPLACE WITH YOUR OWN CONTENT -->
```